docs/API.REST.md

```diff
@@ -697,3 +699,122 @@ Bulk import only works on documents in our persistent data storage layer.
   }
 }
 ```
+
+---
+
+### Getting the last statistics frame
+
+Kuzzle monitors its internal activities and make snapshots of them. This command allows getting the last stored statistics frame.
+
+These statistics include:
+
+* the number of connected users for protocols allowing this notion (websocket, udp, ...)
+* the number of ongoing requests
+* the number of completed requests since the last frame
+* the number of failed requests since the last frame
+
+**URL:** ``http://kuzzle:7512/api/_getStats``
+
+**Method:** ``GET``
+
+**Response:**
+
+```javascript
+{
+  error: null,                      // Assuming everything went well
+  result: {
+    count: <number of found documents>
+    _source: {                      // Your original count query
+      ...
+    },
+    collection: '<data collection>',
+    action: 'getStats',
+    controller: 'admin',
+    statistics: {
+      "YYYY-MM-DDTHH:mm:ss.mmmZ": {
```

Comment:
This was the easiest way I found to be able to easily sort frames when getting multiple statistics frames. And since I decided that getStats and getAllStats should return the same answer structure, here it is.

Another way to format these responses could be to return an array of statistics:

```
{
  error: null,
  result: {
    ...
  statistics: [
    {"timestamp": "YYYY-MM-DDTHH:mm:ss.mmm",   ... statistics informations ...},
    {"timestamp": "YYYY-MM-DDTHH:mm:ss.mmm",   ... statistics informations ...},
    {"timestamp": "YYYY-MM-DDTHH:mm:ss.mmm",   ... statistics informations ...},
    ...
  ],
```